docs/how-to-use-proof.md

```diff
@@ -4,28 +4,27 @@ This document sets out some guidelines for using Asset Proofs in Scalar DL.
 
 ## What is an Asset Proof in Scalar DL?
 
-An Asset Proof in Scalar DL is a set of information about an asset record that is created along with the asset record. It is composed of the following items so that it can be treated as evidence of the creation of an asset record by a ledger.
+An Asset Proof in Scalar DL is a set of information about an asset record and used as evidence of the existence of the asset record. It is composed of the following items.
 
-- The ID of an asset record
+- ID of an asset record
 - Age of the asset record
 - Nonce of the execution request that creates the asset record
 - A cryptographic hash of the asset record
```

Comment:
Maybe it's not necessary but do we need to also mention the previous hash?

Reply:
Thank you! Fixed in [0aaea44](https://github.com/scalar-labs/scalardl/pull/31/commits/0aaea44444e1d49ea84ce6d40a74cd7703df84ae).